### PR TITLE
Make prerender work through gateway

### PR DESCRIPTION
**Pull request by stevenharradine**

### CODE REVIEW

File: defaults/main.yml [hunk 1]
```diff
@@ -22,7 +22,7 @@ nginx_error_log_path: "{{ nginx_log_root }}/error.log"
 nginx_error_log_critical_only: no
 
 nginx_distribution:
-  - nginx-naxsi
+  - "{% if ansible_distribution_major_version|int >= 16 %}nginx{% else %}nginx-naxsi{% endif %}"
```
Comment (aaronpederson) [inline defaults/main.yml]: Wouldn't this be Ubuntu version?

File: templates/etc/nginx/nginx.conf [hunk 2]
```diff
@@ -7,7 +7,7 @@
 
 user {{ nginx_user }};
 worker_processes {{ nginx_worker_processes }};
-pid {{ nginx_pidfile_path }};
+{% if ansible_distribution_major_version|int <= 15 %}pid {{ nginx_pidfile_path }};{% endif %}
```
Comment (aaronpederson) [inline templates/etc/nginx/nginx.conf]: How about this one?

**Comment by stevenharradine** [comment]
Nope ansible_distribution_major_version is correct
http://docs.ansible.com/ansible/playbooks_conditionals.html
```tasks:
  - name: "shut down CentOS 6 and Debian 7 systems"
    command: /sbin/shutdown -t now
    when: (ansible_distribution == "CentOS" and ansible_distribution_major_version == "6") or
          (ansible_distribution == "Debian" and ansible_distribution_major_version == "7")```

**Comment by aaronpederson** [comment]
Team Approved

**Pull request merged**